test/FastTests/Client/LoadAsync.cs

```diff
@@ -60,12 +62,26 @@ public class LoadAsync : RavenTestBase
                     var load1 = session.LoadAsync<User>("users/1");
                     var load2 = session.LoadAsync<User>("users/2");
                     var load3 = session.LoadAsync<User>("users/3");
+                    var load4 = session.LoadAsync<User>("users/4");
+                    var load5 = session.LoadAsync<User>("users/5");
 
                     await Assert.ThrowsAsync<InvalidOperationException>(
                         async () =>
                         {
-                            await Task.WhenAll(load1, load2, load3);
-                        });                   
+                            try
+                            {
+                                await Task.WhenAll(load1, load2, load3, load4, load5);
+                            }
+                            catch (InvalidOperationException)
+                            {
+                                throw;
+                            }
+                            catch (Exception)
+                            {
+                                // we do not have a choice, we are accessing session in a concurrent way
```

Comment:
So this might still fail, right?

Reply:
I wonder if we can make it _not_ race by hitting a slow proxy?

Reply:
Probably not worth it

Reply:
I think we can go with this... and when it fails... remove this test entirely? There are too many places inside Session that can blow when accessed concurrently.